react/components/ProductDescription/index.js

```diff
@@ -12,35 +12,42 @@ import './global.css'
  */
 class ProductDescription extends Component {
   render() {
-    const { specifications, skuName, description } = this.props
+    const { specifications, skuName, description, rowClasses } = this.props
+    let { classes } = this.props
```

Comment:
```suggestion
    const classes = {
      ...ProductDescription.defaultProps.classes,
      ...this.props.classes,
    }
```
This way the re-assignment in line 22 is not necessary